ql/experimental/callablebonds/callablebond.cpp

```diff
@@ -108,6 +109,270 @@ namespace QuantLib {
     }
 
 
+    /* Expose a mechanism to add spread the engine parameters and
```

Comment:
Oops, I forgot to add this to the review.  These classes should probably go into an anonymous namespace.